components/jaeger-agent.libsonnet

```diff
@@ -17,11 +17,21 @@ local containerEnv = container.envType;
     ]) +
     container.mixin.resources.withRequests({ cpu: '32m', memory: '16Mi' }) +
     container.mixin.resources.withLimits({ cpu: '128m', memory: '64Mi' }) +
+    container.mixin.livenessProbe.withFailureThreshold(5) +
```

Comment:
Interesting, were you having problems with single samples? If so, perhaps we need to add this kind of configuration by default as well via Jaeger Operator. Or is it a best-practice to set a threshold to a value like this?

Reply:
They are just my default values, to begin with. Just premature configuration. But if you have any recommended settings, I'd love to take look at them

Reply:
I think that's a good value. We could probably add something similar in the operator, to avoid false-positives.